lib/zendesk_apps_tools/translate.rb

```diff
@@ -95,6 +95,14 @@ def self.source_root
     end
 
     no_commands do
+      def fetch_locale_async(locale, app_package)
+        Thread.new do
+          say("Fetching #{locale['locale']}")
```

Comment:
yeah I was talking to chris and he showed me his technique. So I implemented this.
* Old translate on answer suggestion app: 45.252s
* Fast translate on answer suggestion app: 5.444s